### ISSUE TITLE
Posibility to disable Suggestions

### ISSUE THREAD

**Issue by Xelaa-Dev** [fantomitechno/Emotes#4]
```
public void render(GuiGraphics matrixStack) {
        if (this.suggestions != null && **ModConfig.INSTANCE.show_suggestions**) {
            this.suggestions.render(matrixStack);
        }
        checkTextUpdate();
    }
```
plus whatever needs to be put in the file that geenrates the config

**Comment by fantomitechno** [comment]
It will probably become a non temporary config because it can be very useful... tho idk if the client disable it or the server do or the two can

**Comment by Xelaa-Dev** [comment]
tbh i tried writing it myself but i couldnt figure out how to make it generate the config with the new attribute too rusty in java

**Comment by fantomitechno** [comment]
yeah I use a lib documented but not used on Fabric so it's probably the easiest to find, don't worry, I'll do it like Tuesday or later + it'll probably need a config screen

**Comment by fantomitechno** [comment]
Next commit will add this!

**Comment by fantomitechno** [comment]
42b2303

**Comment by fantomitechno** [comment]
will be in next release after fixes of #3
